src/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+main {
+  width: 96px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.active {
+  background-image: url(../src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.inactive {
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
As I said before, this must be done using :nth-child pseudo class. Check [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) please (remember to use functional notation)